src/legacy/core_plugins/kbn_vislib_vis_types/public/components/options/metrics_axes/value_axes_panel.tsx

```diff
@@ -0,0 +1,162 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import {
+  EuiPanel,
+  EuiTitle,
+  EuiSpacer,
+  EuiAccordion,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiToolTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { BasicVislibParams, ValueAxis } from '../../../types';
+import { ValueAxisOptions } from './value_axis_options';
+import { SetParamByIndex } from './';
+import { ValidationVisOptionsProps } from '../../common';
+
+interface ValueAxesPanelProps extends ValidationVisOptionsProps<BasicVislibParams> {
+  isCategoryAxisHorizontal: boolean;
+  addValueAxis: () => ValueAxis;
+  removeValueAxis: (axis: ValueAxis) => void;
+  onValueAxisPositionChanged: (index: number, value: ValueAxis['position']) => void;
+  setParamByIndex: SetParamByIndex;
+}
+
+function ValueAxesPanel(props: ValueAxesPanelProps) {
+  const { stateParams, addValueAxis, removeValueAxis } = props;
+
+  const getSeries = useCallback(
+    (axis: ValueAxis) => {
+      const isFirst = stateParams.valueAxes[0].id === axis.id;
+      const series = stateParams.seriesParams.filter(
+        serie => serie.valueAxis === axis.id || (isFirst && !serie.valueAxis)
+      );
+      return series.map(serie => serie.data.label).join(', ');
+    },
+    [stateParams.valueAxes, stateParams.seriesParams]
+  );
+
+  const removeButtonTooltip = useMemo(
+    () =>
+      i18n.translate('kbnVislibVisTypes.controls.pointSeries.valueAxes.removeButtonTooltip', {
+        defaultMessage: 'Remove Y axis',
+      }),
+    []
+  );
+
+  const renderRemoveButton = useCallback(
+    (axis: ValueAxis) => (
+      <EuiToolTip position="bottom" content={removeButtonTooltip}>
+        <EuiButtonIcon
+          color="danger"
+          iconType="cross"
+          onClick={() => removeValueAxis(axis)}
+          aria-label={removeButtonTooltip}
+        />
+      </EuiToolTip>
+    ),
+    [removeValueAxis]
+  );
+
+  const addButtonTooltip = useMemo(
+    () =>
+      i18n.translate('kbnVislibVisTypes.controls.pointSeries.valueAxes.addButtonTooltip', {
+        defaultMessage: 'Add Y axis',
+      }),
+    []
+  );
+
+  const getButtonContent = useCallback(
+    (axis: ValueAxis) => {
+      const description = getSeries(axis);
+
+      return (
+        <>
+          {axis.name}{' '}
+          <EuiToolTip content={description}>
+            <>{description}</>
+          </EuiToolTip>
+        </>
+      );
+    },
+    [getSeries]
+  );
+  return (
+    <EuiPanel paddingSize="s">
+      <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween" alignItems="baseline">
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xs">
+            <h2>
+              <FormattedMessage
+                id="kbnVislibVisTypes.controls.pointSeries.valueAxes.yAxisTitle"
+                defaultMessage="Y-Axes"
```

Comment:
It's `Add Y axis` (without a dash), but `Y-Axes` (with a dash and capitalized) and (in another file) `X-axis` (dash and lowercase) - should probably be consistent, not sure which version is the best though - I would vote for `Y axis` / `X axis` without too much thought.

Reply:
nice catch! 
Since `Y-axis`/`X-axis` have already been used on Data tab, I would leave the same version on `Metrics & Axes` tab.

![image](https://user-images.githubusercontent.com/11524637/64541999-7581b500-d32b-11e9-9fad-49807418e07b.png)

Reply:
It should be capital letter with dash and lowercase always `Y-axis`